x/pairing/keeper/staking.go

```diff
@@ -182,13 +182,6 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, validator, creator, chainID strin
 		DelegateCommission: delegationCommission,
 	}
 
-	if stakeEntry.EffectiveStake().LT(spec.MinStakeProvider.Amount) {
```

Comment:
can you add checks to frozen state in stake entry unit tests?

Reply:
done